she_codes_news/users/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
+from django.urls import reverse_lazy
+from django.views.generic.edit import CreateView
+from django.views import generic
+from .models import CustomUser
+from .forms import CustomUserCreationForm
+from django.contrib.auth import get_user_model
+
+class CreateAccountView(CreateView):
+    form_class = CustomUserCreationForm
+    success_url = reverse_lazy('login')
+    template_name = 'users/createAccount.html'
+
+class UserProfileView(generic.DetailView):
+    model = CustomUser
+    slug_field = "username"
+    template_name = 'users/profile.html'
```

Comment:
very nice! 
your userprofile and your authorprofile could well have been the same page, but i like that there are differences here, enought that makes sense to split them